test/bigtest/interactors/package-search.js

```diff
@@ -33,7 +33,10 @@ import SearchBadge from './search-badge';
   hasErrors = isPresent('[data-test-query-list-error="packages"]');
   errorMessage = text('[data-test-query-list-error="packages"]');
   noResultsMessage = text('[data-test-query-list-not-found="packages"]');
-  selectedSearchType = collection('[data-test-search-form-type-switcher] a[class^="is-active--"]');
+  selectedSearchType = collection(
+    '[data-test-search-form-type-switcher] a[class^="primary--"], [data-test-search-form-type-switcher] a[class*=" primary--"]'
```

Comment:
Just to cleanup this 125+ long line could we use `[...].join(', ')` and split the selectors up?